src/Compilers/CSharp/Test/Syntax/Parsing/DeclarationParsingTests.cs

```diff
@@ -5791,6 +5791,213 @@ unsafe struct s
             Assert.Equal(0, file.Errors().Length);
         }
 
+        [Fact]
+        public void CS0071_01()
+        {
+            UsingTree(@"
+public interface I2 { }
+public interface I1
+{
+    event System.Action I2.P10;
+}
+");
+            N(SyntaxKind.CompilationUnit);
+            {
+                N(SyntaxKind.InterfaceDeclaration);
+                {
+                    N(SyntaxKind.PublicKeyword);
+                    N(SyntaxKind.InterfaceKeyword);
+                    N(SyntaxKind.IdentifierToken);
+                    N(SyntaxKind.OpenBraceToken);
+                    N(SyntaxKind.CloseBraceToken);
+                }
+                N(SyntaxKind.InterfaceDeclaration);
+                {
+                    N(SyntaxKind.PublicKeyword);
+                    N(SyntaxKind.InterfaceKeyword);
+                    N(SyntaxKind.IdentifierToken);
+                    N(SyntaxKind.OpenBraceToken);
+                    N(SyntaxKind.EventDeclaration);
+                    {
+                        N(SyntaxKind.EventKeyword);
+                        N(SyntaxKind.QualifiedName);
```

Comment:
>N(SyntaxKind.QualifiedName); [](start = 24, length = 28)

It looks like containment isn't accurately represented by this base-line, which makes it hard to reason about whether it matches expectations. For example, I think the following five nodes are actually children of the QualifiedName. Like so,
```
N(SyntaxKind.QualifiedName);
{
    N(SyntaxKind.IdentifierName);
    {
        N(SyntaxKind.IdentifierToken);
    }
    N(SyntaxKind.DotToken);
    N(SyntaxKind.IdentifierName);
    {
        N(SyntaxKind.IdentifierToken);
    }
}
```

The same goes for the ```N(SyntaxKind.ExplicitInterfaceSpecifier)``` below.

Consider making adjustments to reflect nesting of nodes. If you uncomment the ```//#define PARSING_TESTS_DUMP``` line at the beginning of src\Compilers\CSharp\Test\Syntax\Parsing\ParsingTests.cs and run the test, the test output will contain a dump of the tree with nesting reflected, an indentation will have to be added though.